test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -241,7 +241,7 @@ def test_framework_version_cpu(image):
         else:
             if "neuron" in image:
                 assert tag_framework_version in output.stdout.strip()
-            if all(_string in image for _string in ["pytorch", "habana", "synapseai1.3.0"]):
+            if all(_string in image for _string in ["pytorch", "habana", "synapseai1.4.1"]):
```

Comment:
In the interest of time, lets get this PR merged and then open a parallel PR to make this change backward compatible